File: reto12.js
class Palindromo{
    constructor(palabra){
        this.palabra = palabra
    }
    esPalindromo(){
        let reversedWord = this.palabra.split("").reverse().join()
        console.log(reversedWord)
        console.log(this.palabra)
        return reversedWord == this.palabra
    }
}
let palabra1 = new Palindromo("neuquen")

palabra1.esPalindromo()

File: reto12.py
class Palindromo():
    def esPalindromo(self,palabra):
        reversedWord = "".join(list(reversed(palabra)))
        return reversedWord.lower() == palabra.lower()


palabra1 = Palindromo()
print(palabra1.esPalindromo("hola aloh"))
